[Bug] Xamarin.Forms iOS 14 - 'SensorKit' not found error during debug build/launch

### Description

Greetings, 

After bumping up the Minimum System Version (`MinimumOSVersion`) from `13.6` to `14.0` and updating an iPad to iOS 14 we're unable to build/launch our application in debug mode from Visual Studio for Mac. The code compiles, builds, deploys, and launches without issue but quickly crashes as soon as it begins launching on the device. The app splash screen flashes up for a fraction of a second, and then it dies instantly. There are no exceptions thrown or any other visible output in the IDE I can see to indicate an error. The IDE remains in debug mode with `Waiting for the debugger to connect to Mark's iPad on port ...`

The device log does report the issue which seems to point to a missing SensorKit framework: ` '/System/Library/Frameworks/SensorKit.framework/SensorKit' not found`

```
Incident Identifier: 45EB9EFA-B987-40C7-BE49-E150D7AFC983
CrashReporter Key:   1b617e530dfdeb11cc71e112e25d5095737c4d78
Hardware Model:      iPad7,5
Process:             App.iOS [387]
Path:                /private/var/containers/Bundle/Application/06EEA9D8-A820-4847-8AD8-15BBD4CB0510/App.app/App.iOS
Identifier:          com.company.app
Version:             14.0.10310.0 (5.0.0)
Code Type:           ARM-64 (Native)
Role:                Foreground
Parent Process:      launchd [1]
Coalition:           com.app [529]

Date/Time:           2020-10-19 14:31:05.2812 -0300
Launch Time:         2020-10-19 14:31:05.1731 -0300
OS Version:          iPhone OS 14.0.1 (18A393)
Release Type:        User
Baseband Version:    n/a
Report Version:      104 

Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Description: DYLD, dyld: Using shared cache: 576E48BE-CF7D-3813-BCF6-3D1EFD8C96D3 | dependent dylib '/System/Library/Frameworks/SensorKit.framework/SensorKit' not found for '/private/var/containers/Bundle/Application/06EEA9D8-A820-4847-8AD8-15BBD4CB0510/app/app.iOS', tried but didn't find: '/System/Library/Frameworks/SensorKit.framework/SensorKit'
Highlighted by Thread:  0

Backtrace not available

Unknown thread crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000006   x1: 0x0000000000000001   x2: 0x000000016b9c1f50   x3: 0x00000000000000d9
    x4: 0x000000016b9c1b50   x5: 0x0000000000000000   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x0000000000000020   x9: 0x0000000000000009  x10: 0x6d6172662e74694b  x11: 0x65532f6b726f7765
   x12: 0x694b726f736e6553  x13: 0x77656d6172662e74  x14: 0x736e65532f6b726f  x15: 0x00202774694b726f
   x16: 0x0000000000000209  x17: 0x00000000000000bc  x18: 0x0000000000000000  x19: 0x0000000000000000
   x20: 0x000000016b9c1b50  x21: 0x00000000000000d9  x22: 0x000000016b9c1f50  x23: 0x0000000000000001
   x24: 0x0000000000000006  x25: 0x000000016b9c2818  x26: 0x000000010c0fc778  x27: 0x000000010c0fc000
   x28: 0x0000000000000000   fp: 0x000000016b9c1b20   lr: 0x000000010c0e0394
    sp: 0x000000016b9c1ae0   pc: 0x000000010c0d93e8 cpsr: 0x00000000
   esr: 0x00000000  Address size fault 

Binary images description not available 

Error Formulating Crash Report:
Failed to create CSSymbolicatorRef - corpse still valid ¯\_(ツ)_/¯`
```

### Steps to Reproduce

1. Change `MinimumOSVersion` in info.plist from `13.6` to `14.0` within our existing project.

### Expected Behavior

IDE builds and launches the application on the device as expected.

### Actual Behavior

The application dies quickly when launched in debug.

### Basic Information

- Version with issue: Xamarin.Forms 4.8.0.1560
- Last known good version: Xamarin.Forms 4.8.0.1560
- IDE: Visual Studio for Mac 8.7.8 (build 4)
- Platform Target Frameworks: 
  - iOS:  14.0.0.0
   
- Affected Devices:
iPad (6th gen) - 14.1

### Workaround

Rolling back `MinimumOSVersion` in info.plist from `14.0` to `13.6`.

My bad. I believe that this is the build log. 

Build started...
1>------ Build started: Project: HBSApp.iOS, Configuration: Debug iPhone ------
1>  Executing SayHello Task to establish a connection to a Remote Server. 
1>  			Properties: 
1>  				SessionId=06c0cd9f4da8c8d47c7818505f44da0b92c751360fe90c28d1fb63799bafea80, 
1>  				Addresss=192.168.2.233, 
1>  				SshPort=22, 
1>  				TcpPort=59469, 
1>  				User=Zachary Bulleit, 
1>  				AppName=HBSApp.iOS,
1>  				VisualStudioProcessId=51804,
1>  				ContinueOnDisconnected=False
1>  Detected signing identity:
1>          
1>    Code Signing Key: "Apple Development: Zachary Bulleit (WA7GGBLALN)" (3BDD035BACFEFB545D0176E9203FC01F6C71C984)
1>    Provisioning Profile: "VS: WildCard Development" (90634a16-8078-4387-ad74-8c7496260827)
1>    Bundle Id: com.companyname.HBSApp
1>    App Id: XYDV6T748U.com.companyname.HBSApp
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/icon_about.png obj/iPhone/Debug/optimized/icon_about.png 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/icon_about@2x.png" "obj/iPhone/Debug/optimized/icon_about@2x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/icon_about@3x.png" "obj/iPhone/Debug/optimized/icon_about@3x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/icon_feed.png obj/iPhone/Debug/optimized/icon_feed.png 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/icon_feed@2x.png" "obj/iPhone/Debug/optimized/icon_feed@2x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/icon_feed@3x.png" "obj/iPhone/Debug/optimized/icon_feed@3x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/xamarin_logo.png obj/iPhone/Debug/optimized/xamarin_logo.png 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/xamarin_logo@2x.png" "obj/iPhone/Debug/optimized/xamarin_logo@2x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/xamarin_logo@3x.png" "obj/iPhone/Debug/optimized/xamarin_logo@3x.png" 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/hbs_ios.png obj/iPhone/Debug/optimized/hbs_ios.png 
1>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/user_circle_solid.png obj/iPhone/Debug/optimized/user_circle_solid.png 
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: Found conflicts between different versions of "System.Numerics.Vectors" that could not be resolved.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: There was a conflict between "System.Numerics.Vectors, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Numerics.Vectors, Version=4.1.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     "System.Numerics.Vectors, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Numerics.Vectors, Version=4.1.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "System.Numerics.Vectors, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Numerics.Vectors.dll].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Numerics.Vectors.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Numerics.Vectors.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             System.Numerics.Vectors
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "System.Numerics.Vectors, Version=4.1.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\system.memory\4.5.4\lib\netstandard2.0\System.Memory.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\system.memory\4.5.4\lib\netstandard2.0\System.Memory.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.memory\4.5.4\lib\netstandard2.0\System.Memory.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\azure.core\1.19.0\lib\netstandard2.0\Azure.Core.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\azure.storage.blobs\12.10.0\lib\netstandard2.0\Azure.Storage.Blobs.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\azure.storage.common\12.9.0\lib\netstandard2.0\Azure.Storage.Common.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.harfbuzz\2.88.0-preview.178\lib\netstandard2.0\SkiaSharp.HarfBuzz.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\sqlite-net-pcl\1.8.116\lib\netstandard2.0\SQLite-net.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\sqlitepclraw.core\2.0.4\lib\netstandard2.0\SQLitePCLRaw.core.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\sqlitepclraw.bundle_green\2.0.4\lib\Xamarin.iOS10\SQLitePCLRaw.batteries_v2.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\sqlitepclraw.provider.dynamic_cdecl\2.0.4\lib\netstandard2.0\SQLitePCLRaw.provider.dynamic_cdecl.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\svg.custom\0.5.12\lib\netstandard2.0\Svg.Custom.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\svg.model\0.5.12\lib\netstandard2.0\Svg.Model.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\svg.skia\0.5.12\lib\netstandard2.0\Svg.Skia.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.diagnostics.diagnosticsource\4.6.0\lib\netstandard1.3\System.Diagnostics.DiagnosticSource.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.memory.data\1.0.2\lib\netstandard2.0\System.Memory.Data.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.net.http.json\6.0.0\lib\netstandard2.0\System.Net.Http.Json.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.text.encodings.web\6.0.0\lib\netstandard2.0\System.Text.Encodings.Web.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.text.json\6.0.0\lib\netstandard2.0\System.Text.Json.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\system.text.json\6.0.0\lib\netstandard2.0\System.Text.Json.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\system.text.json\6.0.0\lib\netstandard2.0\System.Text.Json.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.text.json\6.0.0\lib\netstandard2.0\System.Text.Json.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\system.net.http.json\6.0.0\lib\netstandard2.0\System.Net.Http.Json.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: Found conflicts between different versions of "SkiaSharp.Views.iOS" that could not be resolved.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: There was a conflict between "SkiaSharp.Views.iOS, Version=1.68.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" and "SkiaSharp.Views.iOS, Version=2.80.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     "SkiaSharp.Views.iOS, Version=1.68.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" was chosen because it was primary and "SkiaSharp.Views.iOS, Version=2.80.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" was not.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "SkiaSharp.Views.iOS, Version=1.68.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" [C:\Users\Zbull\.nuget\packages\skiasharp.views\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.iOS.dll].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\skiasharp.views\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.iOS.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\skiasharp.views\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.iOS.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.views\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.iOS.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "SkiaSharp.Views.iOS, Version=2.80.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" [].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ImageEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ImageEditor.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ImageEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Map.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Map.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Map.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: Found conflicts between different versions of "System.Net.Http" that could not be resolved.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277: There was a conflict between "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Net.Http, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Net.Http, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Net.Http.dll].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Net.Http.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.Net.Http.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             System.Net.Http
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\microsoft.identity.client\4.42.0\lib\xamarinios10\Microsoft.Identity.Client.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.communitytoolkit\2.0.0\lib\xamarinios10\Xamarin.CommunityToolkit.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.iOS.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Chart.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Common.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Barcode.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ConversationalUI.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataControls.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataGrid.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataVisualization.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ImageEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Input.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Map.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.PdfViewer.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Primitives.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.RichTextEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.SkiaSharp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Core.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Xaml.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\microsoft.identity.client\4.42.0\lib\xamarinios10\Microsoft.Identity.Client.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\microsoft.identity.client\4.42.0\lib\xamarinios10\Microsoft.Identity.Client.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\microsoft.identity.client\4.42.0\lib\xamarinios10\Microsoft.Identity.Client.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\xamarin.communitytoolkit\2.0.0\lib\xamarinios10\Xamarin.CommunityToolkit.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\xamarin.communitytoolkit\2.0.0\lib\xamarinios10\Xamarin.CommunityToolkit.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.communitytoolkit\2.0.0\lib\xamarinios10\Xamarin.CommunityToolkit.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.iOS.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.iOS.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.iOS.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\skiasharp.views.forms\1.68.0\lib\Xamarin.iOS\SkiaSharp.Views.Forms.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Chart.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Common.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Barcode.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ConversationalUI.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataControls.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataGrid.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.DataVisualization.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.ImageEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Input.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Map.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.PdfViewer.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.Primitives.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.RichTextEditor.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\telerik.ui.for.xamarin\2022.1.222.1\lib\xamarinios10\Telerik.XamarinForms.SkiaSharp.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.communitytoolkit\2.0.0\lib\xamarinios10\Xamarin.CommunityToolkit.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Platform.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Core.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\xamarin.forms\5.0.0.2337\lib\Xamarin.iOS10\Xamarin.Forms.Xaml.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:     References which depend on "System.Net.Http, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:         C:\Users\Zbull\.nuget\packages\windowsazure.storage\9.3.3\lib\netstandard1.3\Microsoft.WindowsAzure.Storage.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\Zbull\.nuget\packages\windowsazure.storage\9.3.3\lib\netstandard1.3\Microsoft.WindowsAzure.Storage.dll".
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\.nuget\packages\windowsazure.storage\9.3.3\lib\netstandard1.3\Microsoft.WindowsAzure.Storage.dll
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3277:             C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\bin\Debug\netstandard2.0\HBSApp.dll
1>  Consider app.config remapping of assembly "Microsoft.Bcl.AsyncInterfaces, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "1.0.0.0" [] to Version "6.0.0.0" [C:\Users\Zbull\.nuget\packages\microsoft.bcl.asyncinterfaces\6.0.0\lib\netstandard2.1\Microsoft.Bcl.AsyncInterfaces.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "Newtonsoft.Json, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" from Version "10.0.0.0" [] to Version "13.0.0.0" [C:\Users\Zbull\.nuget\packages\newtonsoft.json\13.0.1\lib\netstandard2.0\Newtonsoft.Json.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "SkiaSharp, Culture=neutral, PublicKeyToken=0738eb9f132ed756" from Version "2.80.0.0" [] to Version "2.88.0.0" [C:\Users\Zbull\.nuget\packages\skiasharp\2.88.0-preview.178\lib\xamarinios1.0\SkiaSharp.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Runtime.CompilerServices.Unsafe, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.4.1" [] to Version "6.0.0.0" [C:\Users\Zbull\.nuget\packages\system.runtime.compilerservices.unsafe\6.0.0\lib\netstandard2.0\System.Runtime.CompilerServices.Unsafe.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Text.Json, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "4.0.0.0" [] to Version "6.0.0.0" [C:\Users\Zbull\.nuget\packages\system.text.json\6.0.0\lib\netstandard2.0\System.Text.Json.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "netstandard, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "2.0.0.0" [] to Version "2.1.0.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\netstandard.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Runtime, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.20.0" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Threading.Tasks.Extensions, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "4.2.0.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Threading.Tasks.Extensions.dll] to Version "4.2.0.1" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Threading.Tasks, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Threading.Tasks.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.IO, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Diagnostics.Debug, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Diagnostics.Debug.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Reflection, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Collections, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Collections.dll] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Runtime.Extensions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Text.RegularExpressions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
1>  Consider app.config remapping of assembly "System.Globalization, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Globalization.dll] to solve conflict and get rid of warning.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2203,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
1>C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp.iOS\Main.cs(17,13,17,58): warning CS0618: 'UIApplication.Main(string[]?, string?, string?)' is obsolete: 'Use the overload with 'Type' instead of 'String' parameters for type safety.'
1>C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\HBSApp.csproj : warning NU1701: Package 'Microsoft.Bcl 1.1.6' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
1>C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\HBSApp.csproj : warning NU1701: Package 'Microsoft.Bcl.Async 1.0.165' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
1>C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp\HBSApp.csproj : warning NU1701: Package 'PCLStorage 1.0.2' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
1>  HBSApp.iOS -> C:\Users\Zbull\Documents\Projects\HBSApp\HBSApp.iOS\bin\iPhone\Debug\HBS.exe
1>  /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/bin/mtouch @/Users/zbulleit/Library/Caches/Xamarin/mtbs/builds/HBSApp.iOS/06c0cd9f4da8c8d47c7818505f44da0b92c751360fe90c28d1fb63799bafea80/obj/iPhone/Debug/response-file.rsp --weak-framework SensorKit -v -v -v -v 
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Xamarin\iOS\Xamarin.iOS.Common.targets(254,3): warning MT0178: Debugging symbol file for '/Users/zbulleit/Library/Caches/Xamarin/mtbs/builds/HBSApp.iOS/06c0cd9f4da8c8d47c7818505f44da0b92c751360fe90c28d1fb63799bafea80/bin/iPhone/Debug/Microsoft.WindowsAzure.Storage.dll' is not valid and was ignored.
1>  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/dsymutil -num-threads 4 -z -o bin/iPhone/Debug/HBS.app.dSYM bin/iPhone/Debug/HBS.app/HBS 
1>  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -i -s obj/iPhone/Debug/mtouch-symbols.list bin/iPhone/Debug/HBS.app/HBS 
1>  /usr/bin/mdimport bin/iPhone/Debug/HBS.app/../ 
1>  /usr/bin/codesign --verify -vvvv "-R=anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.1] exists and (certificate leaf[field.1.2.840.113635.100.6.1.2] exists or certificate leaf[field.1.2.840.113635.100.6.1.4] exists)" bin/iPhone/Debug/HBS.app 
========== Build: 1 succeeded, 0 failed, 2 up-to-date, 0 skipped ==========

--- a/tools/common/Frameworks.cs
+++ b/tools/common/Frameworks.cs
@@ -428,7 +428,7 @@ public class Frameworks : Dictionary<string, Framework> {
 				{ "MLCompute", "MLCompute", new Version (14,0), NotAvailableInSimulator },
 				{ "NearbyInteraction", "NearbyInteraction", 14,0 },
 				{ "ScreenTime", "ScreenTime", 14,0 },
-				{ "SensorKit", "SensorKit", 14,0 },
+				{ "SensorKit", "SensorKit", new Version (14, 0), null, true }, /* not always present on device, e.g. any iPad, so must be weak linked; https://github.com/xamarin/xamarin-macios/issues/9938 */
 				{ "UniformTypeIdentifiers", "UniformTypeIdentifiers", 14,0 },
 
 				{ "AdServices", "AdServices", 14,3 },

